docs/skinning.md

```diff
@@ -0,0 +1,116 @@
+# Skinning Your uPortal-home #
```

Comment:
Is this specific to uPortal-home? or any uPortal-app-framework application you refer to on line 5?